### PR TITLE
child_process: add tests for env

### PR DESCRIPTION
**Pull request by yorkie**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added

Just added the test case for env setting.

### CODE REVIEW

File: test/node/parallel/test-child-process-env.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  'EMPTY': ''
+};
+Object.setPrototypeOf(env, {
+  'FOO': 'BAR'
+});
+
+var child;
+if (false) {
+  child = spawn('cmd.exe', ['/c', 'set'],
+                Object.assign({}, process.env, { env: env }));
```
Comment (legendecas) [inline test/node/parallel/test-child-process-env.js]: Do you mean `{ env: Object.assign({}, process.env, env) }`?

File: test/node/parallel/test-child-process-env.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+} else {
+  child = spawn('/usr/bin/env', [],
+                Object.assign({}, process.env, { env: env }));
```
Comment (legendecas) [inline test/node/parallel/test-child-process-env.js]: Ditto

File: test/node/parallel/test-child-process-env.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  'UNDEFINED': undefined,
+  'NULL': null,
+  'EMPTY': ''
+};
+Object.setPrototypeOf(env, {
+  'FOO': 'BAR'
+});
+
+var child;
+if (false) {
```
Comment (legendecas) [inline test/node/parallel/test-child-process-env.js]: What does `if (false)` mean here?

File: test/node/parallel/test-child-process-env.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+});
+
+process.on('exit', function() {
+  assert.ok(response.search('HELLO=WORLD') > -1);
+  assert.ok(response.search('FOO=BAR') > -1);
+  assert.ok(response.search('UNDEFINED=undefined') > -1);
+  assert.ok(response.search('NULL=null') > -1);
+  assert.ok(response.search(`EMPTY=${os.EOL}`) > -1);
+  console.log('done');
+});
```
Comment (legendecas) [inline test/node/parallel/test-child-process-env.js]: missing EOF

File: test/node/parallel/test-child-process-env.js (hunk 3 above)
Comment (yorkie) [reply test/node/parallel/test-child-process-env.js]: This is copied from Node.js, previously it's for windows.

**Comment by yorkie** [comment]
Close because of inactivity.

**Pull request closed without merging**